### DISCUSSION TITLE
Switch to @stitches/core

### DISCUSSION THREAD

**Discussion by mzaien** [heroui-inc/heroui#155] [Ideas]
Hello there, I'm really exited about next ui and i know it works with any react app, I know that you made a transition to stitches but the react version of it.
If you do not know there is a framework agnostic [api](https://stitches.dev/docs/framework-agnostic) similar to the react one , this will enable anyone to make a port for other frameworks e.g solid, svelte and so on

**Comment by jrgarciadev** [comment]
Hey @mzaien thanks for your suggestion, I will look into it but I think although we implement the `core` version of `Stitches` NextUI will continue to be depending on `react` so the solution to make NextUI `framework agnostic` would be to split all styles into another package 🤔

**Comment by mzaien** [comment]
That would be super awesome, and will make other framework guys come and implement their own implementation
